mmv1/third_party/terraform/fwprovider/framework_provider_access_token_test.go

```diff
@@ -0,0 +1,254 @@
+package fwprovider_test
```

Comment:
I'm just realizing that we'll want this level of testing for universe_domain in the plugin framework version of the provider eventually. Mainly FYI for @hao-nan-li: This is an example of Sarah going through each auth-related provider variable and adding extensive testing because we missed critical changes in auth behavior the first time we tried using the new plugin framework.
`universe_domain` logic is heavily tied to authentication, so we'll probably also want separate tests like these to make sure it's working correctly with the framework provider.

Reply:
I'm happy to leave that work to @hao-nan-li as I lack context about how to test that feature. I'll include Hao as a reviewer on the muxing fix, as it's possible implementation of universe_domain logic currently differs between the SDK and PF code